src/streamlink/plugins/app17.py

```diff
@@ -0,0 +1,68 @@
+import re
+
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import http, useragents
+from streamlink.stream import HLSStream, RTMPStream
+
+API_URL = "https://api-dsa.17app.co/api/v1/liveStreams/isUserOnLiveStream"
+ROOM_URL = "http://17app.co/share/live/{0}"
+
+_url_re = re.compile(r"http://17app.co/share/(?P<page>[^/]+)/(?P<channel>[^/&?]+)")
```

Comment:
It looks like `17.media` is also a domain name used to share links to live streams. You could update the regex to 
`r"http://(17app.co/share|17.media)/(?P<page>[^/]+)/(?P<channel>[^/&?]+)"`